autosubmit/monitor/diagram.py

```diff
@@ -226,29 +225,33 @@ def create_bar_diagram(expid: str, exp_stats: Statistics, jobs_list: List[Job],
     normal_plots_count = 0
     failed_jobs_plots_count = 0
     try:
-        normal_plots_count = int(math.ceil(len(exp_stats.jobs_stat) / MAX_JOBS_PER_PLOT))
-        failed_jobs_plots_count = int(math.ceil(len(exp_stats.failed_jobs) / MAX_JOBS_PER_PLOT))
+        normal_plots_count = int(ceil(len(exp_stats.jobs_stat) / MAX_JOBS_PER_PLOT))
+        failed_jobs_plots_count = int(ceil(len(exp_stats.failed_jobs) / MAX_JOBS_PER_PLOT))
     except Exception as exp:
         Log.warning(str(exp))
 
     # Plotting
     total_plots_count = normal_plots_count + failed_jobs_plots_count
+
+    if total_plots_count == 0:
+        Log.info("The experiment specified does not have any jobs executed.")
+        return False
```

Comment:
This is the second traceback, from my other comment. My experiment had no jobs executed, only jobs in disk, and several files from `inspect`. Now, instead of printing the traceback, the code iwll print this message above, and return `False`, which will prevent AS from a) printing the traceback and b) opening an empty PDF (which over network in ClimateDT can help a lot).